storybookjs/storybook
# Fix loading preact/compat files for preact 8

Issue: When using preact 8 with storybook@next the build fails on require `preact/compat/package.json`. This file is only included in preact10 and not in 8. With preact 8 you'll need to manually install `preact-compat`. 

## What I did
Check for preact version and load files accordingly.

## How to test
load preact 8 instead of 10 with the latest 6.3.0 RC and you'll see the error. Switch to this branch and then everything should be working. 

**Note:** i've never checked for a specific version of a package from node before so I hope this is the correct way to do it.
